README.md

```diff
@@ -51,6 +65,20 @@ export default YourPage
 | style | object | `{}` | Directly set the style object of the svg container |
 | lineStyle | object | `{}` | Set the style object of the line |
 
+### Donut
+
+| Property | Type | Default | Description |
+| - | - | - | - |
+| data | array | `[]` | An array of numbers |
+| width | string | - | CSS width of your chart |
+| height | string | `100%` | CSS height of your chart |
+| colors | string | `CATEGORICAL` | CSS colours used for each data point |
```

Comment:
This should be an array correct? And the default isn't the text `CATEGORICAL`, it's just some default styles

Reply:
Yeah, I wasn't sure how to convey that there is a default set of colours provided
Perhaps `colours.CATEGORICAL` could work? Interested in your take as well

Reply:
Uh, I reckon just some text would work, like the following:

| Property | Type | Default | Description |
| - | - | - | - |
| colors | array | list of 10 sample colors | CSS colours used for each data point |